packages/perseus/src/perseus-types.ts

```diff
@@ -716,14 +716,10 @@ export type MathFormat =
     | "percent"
     | "pi";
 
+export type Simplify = "required" | "correct" | "enforced" | "optional";
```

Comment:
I was going to make this an enum, but then I would've had to update the testdata files to use it as well and I was trying to focus on fixing type errors.